dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Basics/SwitchOff.mo

```diff
@@ -55,6 +55,77 @@ partial model SwitchOffLogic "Manage switch-off logic"
 annotation(preferredView = "text");
 end SwitchOffLogic;
 
+partial model SwitchOffLogicSide1 "Manage switch-off logic for side 1 of a quadripole"
+  /* Handles a predefinite number of switch-off signals and sets running to false as soon as one signal is set to true */
+
```

Comment:
not directly related so should be done in another PR

Reply:
This is a rebase that went wrong in fact

Reply:
Should be fine now.

Reply:
Should be fine now.